phpBB/phpbb/textformatter/s9e/parser.php

```diff
@@ -393,4 +393,49 @@ static public function filter_img_url($url, array $url_config, Logger $logger, $
 
 		return $url;
 	}
+
+	/**
+	* Replace the content displayed inside of a URL tag
+	*
+	* Will only apply to URL tags that do not use any markup (e.g. not "[url]") on the assumption
+	* that those tags were created by the Autolink plugin to linkify URLs found in plain text
+	*
+	* @param  \s9e\TextFormatter\Parser\Tag $url_tag   URL tag (start tag)
+	* @param  \s9e\TextFormatter\Parser     $parser    Parser
+	* @param  string                        $board_url Forum's root URL (with trailing slash)
+	* @return bool                                     Always TRUE to indicate that the tag is valid
+	*/
+	public static function generate_autolink_text(\s9e\TextFormatter\Parser\Tag $url_tag, \s9e\TextFormatter\Parser $parser, $board_url)
+	{
+		// If the tag consumes any text then we ignore it because it's not a linkified URL. Same if
```

Comment:
I'm sorry but this is enough of an extra burden to tip the scale in favor of abandonning this PR. I've spent several hours figuring out how the old code worked and whether it can be made testable that I'm at a point where spending another minute making comments less readable for me is excruciating. I hope you understand.